spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/LocalModule.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.utils
+
+import com.intel.analytics.bigdl._
+import com.intel.analytics.bigdl.dataset.{LocalDataSet, MiniBatch, Sample}
+import com.intel.analytics.bigdl.nn.abstractnn.Activity
+import com.intel.analytics.bigdl.tensor.{Storage, Tensor}
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.optim.LocalPredictor
+
+import scala.reflect.ClassTag
+
+
+object LocalModule {
+  def getAndClearWeightBias[T: ClassTag](parameters: (Array[Tensor[T]], Array[Tensor[T]]))
```

Comment:
@jenniew  There's a `getAndClearWeightBias` under `ModelBroadcast`. Should we refactor and share the function instead of duplicating the code?
and also the weights of the original model would be gone if we use it instead of cloning the origin.

cc @jason-dai